### PR TITLE
Add note for ROS users.

### PR DESCRIPTION
**Pull request by kscottz**
On occasion we receive questions about the maintenance status of Cartographer in ROS. This pull request directs ROS Cartographer users to the correct fork and sets expectations about the maintenance of both this project and its forks.

### CODE REVIEW

**Comment by athackst** [comment]
I'm not opposed to this, but I don't have push access to this repository.  Maybe @MichaelGrupp can help direct?

**Comment by kscottz** [comment]
@athackst thanks. I figured I would ping the familiar faces first. I believe I have admin level access but merging is blocked on a review and I am loathe to turn that off.

**Comment by athackst** [comment]
I see, I didn't know you had admin rights!  That's cool!

One question I have is: Why not just transfer ownership of these repos to ROS? That way folks should get auto-redirected. (or conversely, since you have admin access, add admins to this repo to maintain it here?)

**Review by athackst — APPROVED** [review]
Yeah, doesn't look like I can clear the "review required" hurdle since I don't have write access.

**Review by MichaelGrupp — APPROVED** [review]

**Comment by MichaelGrupp** [comment]
Thanks for adding the note. I do have write access, but since @wohe was doing most of the maintenance in this repo and is one of the original authors, I'll add him to the review.

File: README.rst
Comment (limdor) [inline README.rst:36]: Is this note really only for ROS users or for everyone? The repository seems unmaintained completly.

**Comment by limdor** [comment]
@MichaelGrupp @kscottz @wohe can some of you merge this? It will make life easier to everyone to know that this repository is unmaintained. Thanks

**Comment by kscottz** [comment]
Alright, let's move on from asking for permission, and start begging for forgiveness. I am going to go ahead and merge this.

**Comment by qh-huang** [comment]
What a sad news. However, thank you for the efforts maintaining this project for years.

Is there anything we can do to keep it alive?

**Pull request merged**
